Item/Item.js

```diff
@@ -27,12 +74,36 @@ import Skinnable from '../Skinnable';
  * @ui
  * @public
  */
-const ItemBase = (props) => (
-	<UiItemBase
-		{...props}
-		css={componentCss}
-	/>
-);
+const ItemBase = kind({
+	name: 'Item',
+	styles: {
+		css: componentCss
+	},
+	computed: {
+		className: ({active, selected, styler}) => styler.append({active, selected})
```

Comment:
We may want to consider another prop name than `selected` which overlaps with toggle items like `SwitchItem` and `RadioItem`. It's conceivable we define "selectable items" as different than "toggle items" and implement it by consuming the `selected` prop in toggle items. Or maybe we need to implement this in the base `ToggleItem` and the absence of a "toggle indicator" (e.g. a switch or radio item) we use the border.

What do you think @teckliew and @Djspaceg?

Reply:
We can move both `active` and `selected` to ToggleItem. Although, I'm not sure what the distinction is between `active` and `selected`...
<img width="353" alt="toggleitems" src="https://user-images.githubusercontent.com/8940009/65543734-9ff47600-dec6-11e9-9f08-247d750e15af.png">

Reply:
Does `Item` `selected` differ in purpose/intent from `ToggleItem` `selected`? Curious if we actually need `selected` `itemSelected` and `active`. That seems like a lot of separate states to #1 maintain and #2 for a developer to correctly understand.